module/x/gravity/types/abi_json.go

```diff
@@ -73,4 +73,57 @@ const (
 			{ "internalType": "uint256",   "name": "_invalidationNonce",      "type": "uint256"   }
       ]
     }]`
+
+	DeployERC20ABIJSON = `[{
```

Comment:
Would be nice to expose some keeper methods that create a `ContractCallTx`s with this and the `SendToCosmos` below in it that could be used by governance. Might take a stab at this.